docs/source/about.rst

```diff
@@ -0,0 +1,297 @@
+.. _about:
+
+About
+=====
+
+What is Surround exactly?
+*************************
+
+Surround is an open-source framework developed by the Applied Artificial Intelligence Institute (`A`:superscript:`2`\ `I`:superscript:`2`) to take machine
+learning solutions through from exploration all the way to production. For this reason, it is developed with both
+research engineers and software developers in mind. Designed to play nice with existing machine learning frameworks
+(Tensorflow, MXNet, PyTorch, etc) and cloud services (Google Cloud AI, SageMaker, Rekognition etc), engineers have the freedom
+to use whatever necessary to solve their problem.
+
+A Philosophy
```

Comment:
What do you think about replacing the next four sections with a link to the blog post (once it gets published)? That way this section is lightweight and more about how to use Surround.

Reply:
Yeah would be much better, any idea what the link to it might be once it is published?

Reply:
@ScottyB @zacbrannelly What's the status of this?